src/compiler/diagnosticMessages.json

```diff
@@ -879,7 +879,6 @@
         "category": "Error",
         "code": 1271
     },
-

```

Comment:
the whitespace here is a little useful for pointing out the gap between 1271 and 1300. I'd prefer to keep it and the others that are like it.